app/portainer/services/api/stackService.js

```diff
@@ -4,6 +4,8 @@ function StackServiceFactory($q, Stack, ResourceControlService, FileUploadServic
   'use strict';
   var service = {};
 
+  service.duplicateStack = duplicateStack;
```

Comment:
Please keep the old service function declaration here to avoid a mix of patterns.